CYOAfeb21

```diff
@@ -0,0 +1,225 @@
+if player.playclass== 'A':
+	player.playclass='Warrior'
+	player.items= 'Sword'
+	player.attack= 5
+	player.defense= 10
+	player.health= 15
+	player.magic= 'None'
+elif player.playclass=='B':
+	player.playclass='Wizard'
+	player.items='Wand'
+	player.attack= 10
+	player.defense= 5
+	player.health= 10
+	player.magic= 'Fire'
+elif player.playclass=='C':
+	player.playclass='Cleric'
+	player.items='Relic'
+	player.attack= 4
+	player.defense= 10
+	player.health= 12
+	player.magic= 'Heal'
+elif player.playclass=='D':
+	player.playclass='Rogue'
+	player.items='Dagger' +' ' + 'Lockpick'
+	player.attack= 8
+	player.defense= 8
+	player.health= 10
+	player.magic= 'None'
+else:
+		None
+
+#Function to level up player#
+def levelup():
+		check=player.level
+		if player.experience<5:
+			player.level= 1
+		elif player.experience<10:
+			player.level= 2
+		elif player.experience<20:
+			player.level= 3
+		elif player.experience>20:
+			player.level= 4
+		else:
+			None
+		if player.level != check:
+			print("You leveled up!")
+		else:
+			None
+		
+#Function to call player statblock
+def playerstatblock():
+	self= player
+	statblock=print(f"""
+	Name: {self.name}
+	Position: {self.position}
+	Items: {self.items}
+	Attack: {self.attack}
+	Defense: {self.defense}
+	Health: {self.health}
+	Magic: {self.magic}
+	Experience: {self.experience}
+	Level: {self.level}
+	Class: {self.playclass}""")
+	return self
+playerstatblock()
+
+###Enemies given classes that will later be associated with different outcomes based on what stats the player has.###
+
+###Initiate Battler minotaur###
+minotaur= Battler('Minotaur', "Dungeon", 'Health Potion', 12, 6, 20, 'None', 5, 1, 'Beast')
+def minotaurstatblock():
+	self= minotaur
+	statblock=print(f"""
+	Name: {self.name}
+	Position: {self.position}
+	Items: {self.items}
+	Attack: {self.attack}
+	Defense: {self.defense}
+	Health: {self.health}
+	Magic: {self.magic}
+	Experience: {self.experience}
+	Level: {self.level}
+	Class: {self.playclass}""")
+#minotaurstatblock()
+
+###Function for performing a battle with the minotaur.###
+def minotaurbattle():
+	print("A minotaur blocks your path. You must fight or flee")
+	choice=multich(ch1='Fight', ch2='Flee')
+	if choice=='A':
+		print("You decide to fight.")
+		player.items= player.items + ' ' + minotaur.items
+		minotaur.items= None
+		player.experience= player.experience + minotaur.experience
+		minotaur.health= minotaur.health - (player.attack - minotaur.defense)
+		if player.playclass== 'Wizard':
+			print(f"Your {player.magic} magic defeats the minotaur. You gain {minotaur.experience} experience.")
+		elif player.playclass!='Wizard':
+			print(f"You fight the minotaur and take {minotaur.attack - player.defense} damage.")
+			player.health= player.health - (minotaur.attack - player.defense)		
+#Define choice B		
+	elif choice=='B':
+		print("You decided to flee.")
+	else:
+		None
+	return choice
+
+def mino_encounter():
```

Comment:
You can just call these at the global after the refactor.


```
if decides_to_fight:
  battle(attacker = attacker, defender = defender)
  attacker.levelup()
  attacker.statblock

Reply:
I think you mean I can generalize everything I need to and then copy past that at the end to generate several encounters in a row.